contracts/Splitter.sol

```diff
@@ -2,39 +2,49 @@
 pragma solidity >= 0.5.0 < 0.7.0;
 
 contract Splitter {
```

Comment:
As of now, your contract is completely autonomous. Did you think about reintroducing some centralisation where an _owner_ has some power to pause and resume operations?

Reply:
Yes. I have extended from the Pausable contract.